### PR TITLE
symlinks in examples and scripts

### PR DESCRIPTION
**Pull request by szha**
## Description ##
support symlinks in examples and scripts

### Essentials ###
- [x] Changes are complete (i.e. I finished coding on this PR)
- [x] All changes have test coverage
- [x] Code is well-documented

### Changes ###
- [x] remove duplicate in NMT tutorials by using symlink.

## Comments ##
- we are using zip for creating download files, which should already dereference symlinks. This PR aims to verify that.

### CODE REVIEW

File: scripts/machine_translation/train_transformer.py [hunk 1]
```diff
@@ -444,51 +282,51 @@ def train():
...
+                                                     bucket_scheme=bucket_scheme)
     logging.info('Test Batch Sampler:\n{}'.format(test_batch_sampler.stats()))
-    test_data_loader = DataLoader(data_test,
-                                  batch_sampler=test_batch_sampler,
-                                  batchify_fn=test_batchify_fn,
-                                  num_workers=8)
+    test_data_loader = gluon.data.DataLoader(data_test,
+                                             batch_sampler=test_batch_sampler,
+                                             batchify_fn=test_batchify_fn,
+                                             num_workers=8)
```
Comment (szhengac) [inline scripts/machine_translation/train_transformer.py]: I think we can also put the code for sampler and dataloader into a separate file, including a function called make_dataloader, which returns train_data_loader, val_data_loader, and test_data_loader.

**Comment by mli** [comment]
Job PR-403/26 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-403/26/index.html

**Comment by szha** [comment]
@leezu I forgot to change it back for another PR. Also, Shuai helped with some changes so I thought I'd leave it as is.

**Comment by leezu** [comment]
No worries, was just surprised to see it

**Pull request merged**